config/index.js

```diff
@@ -17,6 +17,11 @@ var conf = convict({
     format: [ 'dev', 'test', 'stage', 'prod' ],
     env: 'NODE_ENV'
   },
+  geodb: {
+    doc: 'Config for geodb location service',
+    dbPath: path.resolve(__dirname, '../node_modules/fxa-geodb/db/cities-db.mmdb'),
```

Comment:
You _may_ be able to use something like `require.resolve()` here instead and make it 1-2% cleaner.

```js
require.resolve('fxa-geodb/db/cities-db.mmdb')
```

Plus, `require.resolve()` also seems to return a fully qualified path:
```js
var path = require("path");

var foo = path.resolve(__dirname, 'node_modules/fxa-geodb/db/cities-db.mmdb');
console.log(foo); // /Users/pdehaan/dev/github/mozilla/blok/node_modules/fxa-geodb/db/cities-db.mmdb

var bar = require.resolve("fxa-geodb/db/cities-db.mmdb");
console.log(bar); // /Users/pdehaan/dev/github/mozilla/blok/node_modules/fxa-geodb/db/cities-db.mmdb
```